romanian/nalba.md

```diff
@@ -10,4 +10,5 @@ Va dati impreuna capat pana ramaneti fara... pat.
 Dupa ce mancati voi pana dimneata, de vi se umple burtile o apuca pe Gina si striga:
 
 - Auzi mah sti ce m-am gandit?
-
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.